assets/templates/census-landing.tmpl

```diff
@@ -1,6 +1,9 @@
 <div class="ons-page__container ons-container">
     <div class="ons-grid ons-js-toc-container ons-u-ml-no">
         {{ template "partials/breadcrumb" . }}
+        {{ if .Error.Title }}
```

Comment:
Might be wrong but is .Error right? Is it not just Error?

Reply:
It's a prop on the model